File: JavaScript/leetcode_824.js
/**
 * @param {string} sentence
 * @return {string}
 */
var toGoatLatin = function(sentence) {
    const vovels = new Set (['a', 'e', 'i', 'o', 'u']);
    let repeat = 1;
    let flist = [];
    let s = sentence.split(" ");
    for (let index = 0; index < s.length; index++) {
        if (vovels.has(s[index][0].toLowerCase())) {
            let w = s[index] + 'ma' + 'a'.repeat(repeat);
            flist.push(w)
            repeat++
        }
        else {
            let w = s[index].slice(1) + s[index][0] + 'ma' + 'a'.repeat(repeat)
            repeat++;
            flist.push(w)
        }
        
    }
    return flist.join(" ")
};

File: Python/leetcode_824.py
def to_goat_latin(sentence: str) -> str:
    vovels = {'a', 'e', 'i', 'o', 'u'}
    repeat = 1
    f_list = []
    s = sentence.split()
    for w in s:
        if w[0].lower() in vovels:
            w = w + 'ma' + 'a' * repeat
            repeat += 1
            f_list.append(w)
        else:
            w = w[1:] + w[0] + 'ma' + 'a' * repeat
            repeat += 1
            f_list.append(w)
    return " ".join(f_list)


print(to_goat_latin("I speak Goat Latin"))
